pom.xml

```diff
@@ -1024,6 +1024,7 @@
                             <exclude>debian/rules</exclude>
                             <exclude>debian/source/format</exclude>
                             <exclude>dist/console-proxy/js/jquery.js</exclude>
+                            <exclude>engine/schema/dist/**</exclude>
```

Comment:
Added this to exclude, the dist isn't in the source but gets created during maven build.